pkg/kubectl/rolling_updater.go

```diff
@@ -138,11 +149,32 @@ func (r *RollingUpdater) Update(config *RollingUpdaterConfig) error {
 		}
 	}
 
+	if oldRc == nil {
+		// There's no source RC, so just scale up the new immediately and return
```

Comment:
This block identical to the final resize step for the general case (line 208). Should make it a function to avoid code duplication.